postfix/common.sls

```diff
@@ -55,11 +55,24 @@ postfix_aliases:
   - require:
     - pkg: postfix_packages
 
+/var/lib/mailman/data:
```

Comment:
Here should be dependency on mailman packages (because they create list user/group)